### PR TITLE
Disable homebrew update during Travis/OSX build

### PR DESCRIPTION
**Pull request by Xarthisius**
Trying to shave off a few minutes of the OSX build. A lot of time is being spent installing and updating things we're not even using.

### CODE REVIEW

**Comment by Xarthisius** [comment]
This should be good to go as soon as all tests turn green.

File: .travis.yml [hunk 1]
```diff
@@ -30,9 +31,8 @@ before_install:
     else
       sudo mkdir -p /usr/local/man
       sudo chown -R "${USER}:admin" /usr/local/man
-      # brew update  # do not update to save build time
-      brew install python ccache hdf5 proj geos openmpi netcdf
-      brew uninstall gdal postgis numpy
+      HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 proj geos open-mpi netcdf ccache
+      HOMEBREW_NO_AUTO_UPDATE=1 brew uninstall gdal postgis numpy  # WHY?
```
Comment (neutrinoceros) [inline .travis.yml:35]: I think maybe someone should answer this question before we merge 😅

Comment (Xarthisius) [reply .travis.yml:35]: https://github.com/yt-project/yt/commit/6f6894e270bc8889509d7711a545c4ebbfd57b2e
I assume it was throwing some sort of error

**Comment by matthewturk** [comment]
IT'S GREEN

**Pull request merged**